src/__experimental__/components/select/select.spec.js

```diff
@@ -44,7 +45,7 @@ describe('Select', () => {
 
   // utility functions to fetch various elements from the wrapper
   const listOf = wrapper => wrapper.find('SelectList');
-  const textboxOf = wrapper => wrapper.find('Textbox');
+  const textboxOf = wrapper => wrapper.find(TextBox);
   const pillsOf = wrapper => wrapper.find('Pill');
```

Comment:
Is there a reason for the inconsistency in the types of parameters passed to this set of functions? (`'SelectList'` and `'Pill'` are strings, but `TextBox` is a component)

Reply:
We're looking for the TextBox component type and the others are looking for components with the display name of `Pill`, `SelectList`. It was the minimum set of changes required to make the tests pass.

This worked previously because the HOC would append the component display name.
For example `Textbox` would become `WithUniqueName(WithValidations(Textbox)) > WithValidations(Textbox) > Textbox` but for v9 we have chosen to flatten the names so there will be multiple components with the same name (`Textbox > Textbox > Textbox`) due to the amount of GAC tests that are expecting 'Textbox' to return the top-level component. In a future version, we expect to re-instate the HOC display names.

When reviewing this comment I did notice the `Textbox` display name was `ForwardRef` in the snapshots so I fixed that in 2f027d8

* `.find('Textbox')` returns 3 elements
* `.find(TextBox)` returns 1 element

Reply:
@jamime: OK, many thanks for the detailed explanation :thumbsup: